optimum/gptq/quantizer.py

```diff
@@ -635,8 +636,16 @@ def load_quantized_model(
         device_map = {"": torch.cuda.current_device()}
         logger.info("The device_map was not initialized." "Setting device_map to `{'':torch.cuda.current_device()}`.")
 
-    with open(os.path.join(save_folder, quant_config_name), "r", encoding="utf-8") as f:
-        quantize_config_dict = json.load(f)
+    # this branch is to add support for loading quantization_config from config.json
+    if os.path.exists(os.path.join(save_folder, CONFIG_NAME)):
+        with open(os.path.join(save_folder, CONFIG_NAME), "r", encoding="utf-8") as f:
+            try:
+                quantize_config_dict = json.loads(f)["quantization_config"]
+            except KeyError:
```

Comment:
We should not raise an error here. Models that are quantized using auto-gptq don't have the `quantization_config` in the `config`. We should raise an error if both way did not work.